fedora/client/bodhi.py

```diff
@@ -142,7 +142,7 @@ def query(self, release=None, status=None, type_=None, bugs=None,
         auth = False
         if params['mine']:
             auth = True
-        for key, value in params.items():
+        for key, value in list(params.items()):
```

Comment:
Did you see the ``del`` below?  It was raising an error about the dict changing size during iteration (python3 only).